static/js/compose_validate.js

```diff
@@ -444,7 +444,7 @@ function validate_stream_message() {
         const topic = compose_state.topic();
         // TODO: We plan to migrate the empty topic to only using the
         // `""` representation for i18n reasons, but have not yet done so.
-        if (topic === "" || topic === "(no topic)") {
+        if (topic === "" || topic === "general chat") {
```

Comment:
This would result in user not allowed to have "general chat" as topic when realm has set the setting to require topics as `True` and that can confuse the user.

Reply:
@alya thoughts?